rbmc-cfam-daemon/README.md

```diff
@@ -16,3 +16,18 @@ It will read the sibling's CFAM every 2 seconds. A D-Bus interface will be
 created to make these values available to other applications.
 
 The `rbmc-cfam-daemon` meson option will enable building this code.
+
+## CFAM-S ScratchPad Register Layout
+
+| Field            | Register | Start Bit | Length |
+| ---------------- | -------- | --------- | ------ |
+| API Version      | 1        | 0         | 8      |
+| BMC Position     | 1        | 8         | 1      |
+| Role             | 1        | 9         | 2      |
+| Red Enabled      | 1        | 11        | 1      |
+| Failovers Paused | 1        | 12        | 1      |
+| Provisioned      | 1        | 13        | 1      |
+| BMC State        | 1        | 14        | 3      |
+| Sibling Comms OK | 1        | 17        | 1      |
+| Heartbeat        | 1        | 24        | 8      |
```

Comment:
Ideally, I'd like it to be in its own register, so we won't be read/modify/writing a reg with other fields in it constantly.  However, we are really limited with 4 registers so I'd like to preserve as much space as possible for future expansion.  And since this is the only daemon touching that register, there shouldn't be any problems with overwrites or anything.

The plan is incrementing the heartbeat in the local BMC every second, and then having the sibling read it every 2 to 5 seconds (we had discussed 5 in the workshop, but I like 2 better for now).  So it would be unlikely that it would wrap around to the same value again in between reads, such that the other BMC didn't see the change.